lib/StaticNarrative/exporter/preprocessor.py

```diff
@@ -203,9 +219,9 @@ def preprocess_cell(self, cell, resources: dict, index: int):
                 ref = self._get_data_cell_ref(kb_meta, ws_id)
                 if ref is not None:
                     kb_info["external_link"] = f"{self.host}/#dataview/{ref}"
-            cell.metadata["kbase"] = kb_info
         else:
             kb_info = {"type": "nonkb"}
+        cell.metadata["kbase"] = kb_info
```

Comment:
`kb_info` wasn't getting stored in the case where `kb_info` == `{"type": "nonkb"}` so I just moved this call out a level.